mappings/net/minecraft/world/border/WorldBorderListener.mapping

```diff
@@ -1,21 +1,28 @@
 CLASS net/minecraft/class_2780 net/minecraft/world/border/WorldBorderListener
 	METHOD method_11929 onDamagePerBlockChanged (Lnet/minecraft/class_2784;D)V
-		ARG 1 damagePerBlock
+		ARG 1 border
+		ARG 2 damagePerBlock
 	METHOD method_11930 onCenterChanged (Lnet/minecraft/class_2784;DD)V
-		ARG 1 centerX
-		ARG 2 centerZ
+		ARG 1 border
+		ARG 2 centerX
+		ARG 4 centerZ
 	METHOD method_11931 onInterpolateSize (Lnet/minecraft/class_2784;DDJ)V
 		ARG 1 border
 		ARG 2 fromSize
 		ARG 4 toSize
 		ARG 6 time
 	METHOD method_11932 onWarningTimeChanged (Lnet/minecraft/class_2784;I)V
-		ARG 1 warningTime
+		ARG 1 border
+		ARG 2 warningTime
 	METHOD method_11933 onWarningBlocksChanged (Lnet/minecraft/class_2784;I)V
-		ARG 1 warningBlocks
+		ARG 1 border
+		ARG 2 warningBlocks
```

Comment:
Why this name for an integer?

Reply:
this is like the warning block distance, I'd assume?

Reply:
Should be renamed then, this is confusing